crates/pop-parachains/src/utils/git.rs

```diff
@@ -12,6 +13,8 @@ use url::Url;
 
 pub struct Git;
 impl Git {
+	const GIT_SSH: &'static str = "git@github.com:";
```

Comment:
This is a GH specific string rather than git, so suggest renaming to GITHUB_SSH and then reusing it from the ssh repo function.

Reply:
There is a GitHub type somewhere, so maybe its better to be defined there.